data/db/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/UserDataStore.kt

```diff
@@ -32,6 +32,26 @@ abstract class UserDataStore {
         )
     }
 
+    fun favoritesTimetable(): Flow<Set<String>> {
```

Comment:
Since the `favoritesTimetable` represents the entire timetable, how about using `favoriteTimetableItemIds`? 📛

Reply:
That make sense!